client/my-sites/menus/main.jsx

```diff
@@ -292,4 +296,18 @@ var Menus = React.createClass( {
 	}
 } );
 
-module.exports = protectForm( Menus );
+const connectComponent = connect(
+	state => {
+		const siteId = getSelectedSiteId( state );
+		const isRequesting = isRequestingPostTypes( state, siteId );
+		return {
+			isRequesting,
+			siteId,
+		};
+	}
+);
+
+export default flowRight(
+	connectComponent,
```

Comment:
Yeah, I could but, this would mean I'll have to make a lot of changes to refactor using the last coding standards. I'm trying to limit the changes in this PR to the Redux usage only. It's already a big PR don't you think?